StorageTests/TestSQLiteRemoteClientsAndTabs.swift

```diff
@@ -132,18 +132,25 @@ class SQLRemoteClientsAndTabsTests: XCTestCase {
         } catch _ {
         }
         clientsAndTabs = SQLiteRemoteClientsAndTabs(db: BrowserDB(filename: "browser.db", files: files))
+
+        // Workaround: the remote devices table is created in BrowserTable, not in SQLiteRemoteClientsAndTabs.
+        _ = clientsAndTabs.db.createOrUpdate(BrowserTable())
```

Comment:
Would it make more sense to run this prior to line 134, creating the DB, adding the right table, and then making `clientsAndTabs`?